dotCMS/src/main/java/com/dotcms/rest/api/v1/system/ConfigurationHelper.java

```diff
@@ -135,9 +141,29 @@ private ConfigurationHelper() {
 				CLUSTER, clusterMap(user));
 
 	    map.put(LANGUAGES, APILocator.getLanguageAPI().getLanguages());
+	    map.put(TIMEZONES, getTimeZones(locale));
 	    return map;
 	}
 
+    /**
+     * Returns a list of all available timezones
+     * @param locale
+     * @return
+     */
+	private List getTimeZones(final Locale locale){
+
+        final List<Map<String, Object>> timeZoneList = new ArrayList<>();
```

Comment:
great job, I would just add or mod a curl in order just to check that this field is always returned and is length >0